tests/components/workday/test_binary_sensor.py

```diff
@@ -71,7 +131,21 @@ def setup_method(self):
                 "province": "BW",
                 "workdays": ["mon", "wed", "fri"],
                 "excludes": ["sat", "sun", "holiday"],
-                "add_holidays": ["2020-02-24"],
+                "add_holidays": [
+                    date(2020, 2, 24),
+                    [date(2020, 2, 27), date(2020, 3, 2)],
+                ],
```

Comment:
If I follow the logic correct, I think that config should be YYYY-MM-DD strings, and should be parsed as date in the validator.
```suggestion
                "add_holidays": [
                    "2020-02-24",
                    ["2020-02-27", "2020-03-02"],
                ],
```

Reply:
When I exercise the code in HA, the voluptuous schema is called with a date, not a string. So the test reflects that. I also added a code comment about it at https://github.com/mxr/core/blob/69d6fd587cd68631b5dcb97ebdd07c610e77a7dc/homeassistant/components/workday/binary_sensor.py#L65-L66